server/src/main/java/org/tctalent/server/service/db/impl/CandidateLanguageServiceImpl.java

```diff
@@ -210,7 +209,6 @@ public List<CandidateLanguage> updateCandidateLanguages(UpdateCandidateLanguages
                         languageLevels.get(update.getSpokenLevelId())
                 );
             }
-            updatedLanguages.add(candidateLanguageRepository.save(candidateLanguage));
             updatedLanguageIds.add(candidateLanguage.getId());
         }
```

Comment:
I don't think you can just delete this line - it will mean that the language is not updated on the database